api/buckets/service.go

```diff
@@ -116,6 +118,7 @@ func (s *Service) createBucket(ctx context.Context, dbID thread.ID, dbToken thre
 			}
 		}
 	}
+	go s.IPNSManager.publish(pth, buck.Key)
```

Comment:
I think that would mean having a service level context, since using the request contexts are short lived compared to what's needed for publishing. 

There's a `Cancel` method on the ipns manager that was meant to solve this issue, but I forgot to wire it up core's close method.